std/random.d

```diff
@@ -359,12 +359,11 @@ $(D x0).
 /**
    (Re)seeds the generator.
 */
-    void seed(UIntType x0 = 1) @safe pure
+    void seed(UIntType x0 = 1) @safe @nogc pure nothrow
     {
         static if (c == 0)
         {
-            import std.exception : enforce;
-            enforce(x0, "Invalid (zero) seed for "
+            x0 || assert(0, "Invalid (zero) seed for "
```

Comment:
Okay, but why are you `||`ing instead of using `assert(x0, ...)`? That's not how assertions are normally used.

And since you changed it to an assertion, you can make the function `nothrow`.